### ISSUE TITLE
plugin-typescript >= 4.0.9 breaks plugin-json

### ISSUE THREAD

**Issue by andrenarchy** [frankwallis/plugin-typescript#128]
With `plugin-typescript>=4.0.9` my setup fails while loading the json plugin for systemjs. Everything works nicely with `plugin-typescript<=4.0.8`. 

It tries to download the file 
```
http://localhost:8080/localhost:8080/jspm_packages/github/systemjs/plugin-json@0.1.2/json.js
```
but it should look for 
```
http://localhost:8080/jspm_packages/github/systemjs/plugin-json@0.1.2/json.js
```

Any ideas?

**Comment by frankwallis** [comment]
Thank you for raising this, it should be fixed in 4.0.16

**Comment by andrenarchy** [comment]
I can confirm that the issue is fixed, thanks! :+1:
